core/src/main/java/org/apache/accumulo/core/data/Mutation.java

```diff
@@ -1613,22 +1615,22 @@ public void write(DataOutput out) throws IOException {
     }
     out.write((byte) (0x80 | hasValues));
 
-    WritableUtils.writeVInt(out, row.length);
+    UnsynchronizedBuffer.writeVInt(out, integerBuffer, row.length);
```

Comment:
Is this change compatible with the old implementation? If not, we should highlight that any persisted Mutations won't work across versions (if users did that) in the release notes.

Reply:
It should be.  It should be writing the same bytes... i.e. when only 2 bytes from an int get written, those same 2 bytes should get written.

Reply:
would probably be nice to add to the javadocs that the methods are compat with hadoops methods